moveit_core/background_processing/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
 set(MOVEIT_LIB_NAME moveit_background_processing)
 
-add_library(${MOVEIT_LIB_NAME}
-  src/background_processing.cpp
-  )
+add_library(${MOVEIT_LIB_NAME} src/background_processing.cpp)
```

Comment:
while its not worth changing, I prefer the previous format because it makes adding more files to the library list cleaner in the future